cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/openid/casOpenIdServiceFailureView.jsp

```diff
@@ -17,5 +17,4 @@
     specific language governing permissions and limitations
     under the License.
 
---%>
-<%= "openid.mode:id_res\nis_valid:false\n" %>
\ No newline at end of file
+--%><%= "openid.mode:id_res\nis_valid:false\n" %>
```

Comment:
This one is very tricky. How is the output actually rendered? I think the underlying openid lib has issues parsing \n and \r characters.